pwiz_tools/Skyline/Properties/Settings.cs

```diff
@@ -2900,7 +2900,7 @@ public sealed class SrmSettingsList : SerializableSettingsList<SrmSettings>
                         new[] { IonType.y }, // PeptideFragmentTypes
                         new[] { Adduct.M_PLUS_H, }, // SmallMoleculePrecursorCharges
                         new[] { Adduct.M_PLUS, }, // SmallMoleculeProductCharges
-                        Transition.MOLECULE_ION_TYPES, // SmallMoleculeFragmentTypes

```

Comment:
Let's not change the defaults in this pull request. Let's leave it compatible with the peptide defaults, and then we can implement a second change set that adds "p" to this setting when MS1 gets turned on. I don't think people doing SRM want this default precursor targeting, but everyone extracting MS1 chromatograms does, and we have a problem of teaching them to enable MS1 and add "p" to this setting. I don't think making "p" on by default only for small molecules is the solution.

Reply:
Fair enough, leaving it at default "f" makes it consistent with peptide behavior (though adding ", p" on enabling full scan would not be consistent with peptide behavior, so best to leave it alone completely I guess).

Reply:
OK, those changes are in.